docs/Other SDL Features/Retrieving Vehicle Data/index.md

```diff
@@ -184,8 +204,46 @@ sdlManager.send(request: subscribeVehicleData) { (request, response, error) in
     // Successfully subscribed
 }
 ```
+!@
+
+@![android, javaSE, javaEE]
+**First**, you should add a notification listener for the `OnVehicleData` notification: 
+
+```java
+sdlManager.addOnRPCNotificationListener(FunctionID.ON_VEHICLE_DATA, new OnRPCNotificationListener() {
+    @Override
+    public void onNotified(RPCNotification notification) {
+        OnVehicleData onVehicleDataNotification = (OnVehicleData) notification;
+        if (onVehicleDataNotification.getPrndl() != null) {
+            Log.i("SdlService", "PRNDL status was updated to: " + onVehicleDataNotification.getPrndl());
+        }
+    }
+});
+```
+
+**Then**, send the `SubscribeVehicleData` request:
+
+```java
+SubscribeVehicleData subscribeRequest = new SubscribeVehicleData();
+subscribeRequest.setPrndl(true);
+subscribeRequest.setOnRPCResponseListener(new OnRPCResponseListener() {
+    @Override
+    public void onResponse(int correlationId, RPCResponse response) {
+        if(response.getSuccess()){
+            Log.i("SdlService", "Successfully subscribed to vehicle data.");
+        }else{
+            Log.i("SdlService", "Request to subscribe to vehicle data was rejected.");
+        }
+    }
+}); 
+sdlManager.sendRPC(subscribeRequest);
+```
+
+**After that**, the `onNotified` method will be called when there is an update to the subscribed vehicle data.
```

Comment:
Instead of **Then** and **After that**, I'd prefer to see **Second** and **Third**

Reply:
Fixed.